### PR TITLE
Update espresso-sequencer-go with new LC ABI

### LINKED ISSUE #229: Update Light Client ABI
We need to update the interface of the LC contract to match the recent change in the sequencer repo:

https://github.com/EspressoSystems/espresso-sequencer/pull/1939/files#diff-d85261c27bf48fe0ec00e8cc9bb3ca101d708ec68c85968531b2e2cf4f9ed292R409

- [ ] https://github.com/EspressoSystems/espresso-sequencer-go/pull/49
- [ ] https://github.com/EspressoSystems/nitro-espresso-integration/pull/230

### PR DESCRIPTION
**Pull request by sveitser**
Instead of using the block height from the first snapshot after the height in the justification it uses the latest height in the LC contract. Currently not 100% convinced about correctness but also believe it should work because any newer tree still contains the header of interest.

Closes #229

### CODE REVIEW

File: arbnode/batch_poster.go [hunk 1]
```diff
@@ -506,23 +506,28 @@ func (b *BatchPoster) addEspressoBlockMerkleProof(
 			return fmt.Errorf("this msg has not been included in hotshot")
 		}
 
-		snapshot, err := b.lightClientReader.FetchMerkleRoot(jst.Header.Height, nil)
+		_, err = b.lightClientReader.FetchMerkleRoot(jst.Header.Height, nil)
```
Comment (Sneh1999) [inline arbnode/batch_poster.go:509]: Do we need to fetch the merkle root now?

Comment (zacshowa) [reply arbnode/batch_poster.go:509]: It seems that the only thing we needed from the snapshot was the hotshot height. I think were fine to remove this now.

Comment (nomaxg) [reply arbnode/batch_poster.go:509]: Why do we still need this call at all?

**Comment by nomaxg** [comment]
This seems correct because the validated height should be equivalent to the latest snapshot height

**Review by Sneh1999 — APPROVED** [review]

**Pull request merged**